core/src/main/scala/caliban/transformers/Transformer.scala

```diff
@@ -326,20 +328,80 @@ object Transformer {
       }
   }
 
+  object ExcludeDirectives {
+
+    /**
+     * A transformer that allows excluding fields and inputs with specific directives.
+     *
+     * {{{
+     *   case object Experimental extends GQLDirective(Directive("experimental"))
+     *   case object Internal extends GQLDirective(Directive("internal"))
+     *
+     *   ExcludeDirectives(Experimental, Internal)
+     * }}}
+     */
+    def apply(directives: GQLDirective*): Transformer[Any] =
```

Comment:
I wonder if there should also be a filter variant `GQLDirective => Boolean`, since this would require exact match of arguments as well

Reply:
I was actually thinking that as well. I'll add it 👍

Reply:
Done